build/yaml/testScenarios/runScenario.yml

```diff
@@ -0,0 +1,34 @@
+parameters:
+  consumers: []
+  skills: []
+
+stages:
+- ${{ each consumer in parameters.consumers }}:
```

Comment:
Is this still needed given the new implementtion in the branch that generates the test cases using code?

Reply:
This will probably change when we start working with the new implementation of the test project, but we wanted to wrap up what we had to start the next stage over this.

Reply:
OK, let's address this later then.